build.gradle

```diff
@@ -8,7 +8,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:7.2.2'
+        classpath 'com.android.tools.build:gradle:7.4.2'
```

Comment:
Not sure if this is required for the kotlin conversion. Generally avoid updating any dependencies.

Reply:
I'll keep that in mind for the future. Thank you!